xml/System.Threading.Tasks/TaskFactory`1.xml

```diff
@@ -596,7 +596,7 @@
       <Docs>
         <param name="tasks">The array of tasks from which to continue.</param>
         <param name="continuationFunction">The function delegate to execute asynchronously when all tasks in the <paramref name="tasks" /> array have completed.</param>
-        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The <see langword="NotOn*" /> and <see langword="OnlyOn*" /> values are not valid.</param>
+        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The <see langword="NotOn" /> and <see langword="OnlyOn" /> values are not valid.</param>
```

Comment:
We need the * to render. Should it be like this instead or should we keep it as-is because the markdig branch is displaying it correctly?
https://review.docs.microsoft.com/en-us/dotnet/api/System.Threading.Tasks.TaskFactory-1.ContinueWhenAll?branch=markdig-migration&view=netframework-4.8#System_Threading_Tasks_TaskFactory_1_ContinueWhenAll_System_Threading_Tasks_Task___System_Func_System_Threading_Tasks_Task____0__System_Threading_CancellationToken_System_Threading_Tasks_TaskContinuationOptions_System_Threading_Tasks_TaskScheduler_

```suggestion
        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The <see langword="NotOn\*" /> and <see langword="OnlyOn\*" /> values are not valid.</param>
```

Reply:
So let`s keep the content since it shows what we want. Escaping in the html tag may not produce expected result.